introduction.md

```diff
@@ -1,34 +1,79 @@
-- Have you ever wondered how they work or thought about building something like them or doing something with them? 
-- What do you want to become in your life? 
+- Have you ever wondered how they work or thought about building something like them or doing something with them?
+- What do you want to become in your life?
 
-- We should check if our upcoming expames and analogies are relavent to the kids or not. 
+- We should check if our upcoming expames and analogies are relavent to the kids or not.
 
-### Morning Routine 
-Description : An unplugged exercise to show the kids what an algorithm or generic code looks like, by breaking down their morning 
-routine of getting up from bed and going to school. We can introduce code structure, flow, if statements, loops, through this exercise 
+Description : An unplugged exercise to show the kids what an algorithm or generic code looks like
 
-### History of CS 
-Description : Walk the kids through the history of computers, to answer questions like "why were they invented", 
-"what problems do they solve", "why do we need them", "how did they evolve over time". The goal here is to establish the context of why 
-computers exist in the first place, but in a very story-like fashion. 
+Goal: A metaphor that we can map back to coding.
+We can introduce code structure, flow, if statements, loops, through this exercise and keep getting back to this example throughout the course.
+
+For week1, we can stop with using this to introduce the concept of `algorithm`, and maybe
+`code` and `programming language`.
+
+*Example#1*
+- Breaking down their morning routine of getting up from bed and going to school
+What we expect more or less - get up => brush => shower =>
+breakfast => wait for the right bus (loops/conditionals) => get to class =>
+greet your friends (function concept), etc
+
+*Example#2*
+- Planning a birthday party for a friend. <TODO: @Madhu94 add a short description>
+
+*Example#3* (From code.org)
+- Planting a seed and growing a tree
+
+We will encourage students to contribute answers and build from that.
+
+(We need a wide range of examples here, so that we can alter our examples based on the
+results of the survey.)
+
+### History of CS
+Description : Walk the kids through the history of computers, to answer questions like
+- Why were computers invented?
+- How did they evolve over time?
+
+The goal here is to establish the context of why computers exist in the first place, but in a very story-like fashion,
+with slides/pictures/video.
+
+- Analogy between humans solving a problem and computers based on the unplugged exercise.
+This transitions naturally to why we need computers in the first place.
+- Man had aids to computing - abacus, Pascal's adding machine,etc. They aren't computers
+    in that they can't be programmed, they were like tools that helped to make computing
+    easier. Human computers.
+- Babbage's need for a computer to automate navigation log tables. This design was the
```

Comment:
This section might not be good idea to include in the Introduction - History of CS. We don't want to introduce events / concepts (automate, log tables etc) or even people that the kids already don't have any idea about. If we do, then we will have to go into the details of the event/topic/concept or person themselves, which would divert the session unnecessarily. We should try to stick to analogies and examples that the kids are already familiar with to some degree, even when we're going through the history of computers and why they were invented

Reply:
True, will remove. There was no reason to try to be historically accurate here.